mlops/nyc_taxi/src/mlops_pipeline.py

```diff
@@ -229,18 +230,28 @@ def execute_pipeline(
                         or pipeline_job.status == "CancelRequested"
                         or pipeline_job.status == "Canceled"
                     ):
+                        print(
+                            f"Pipeline job '{pipeline_job.name}' has stopped with status: {pipeline_job.status}."
+                        )
```

Comment:
Minor comment.. this line does not appear in london_taxi.

Having these two files be exactly the same will make it easier for when we can move this reused code to another location. I would recommend copy/pasting the updated sections from one file to another so they match.

Reply:
Added this line to london taxi so they match